package.json

```diff
@@ -20,17 +23,26 @@
     "expo-linking": "^6.3.1",
     "expo-router": "^3.5.16",
     "expo-status-bar": "~1.12.1",
+    "postgres": "^3.4.4",
     "react": "18.2.0",
     "react-native": "0.74.2",
     "react-native-safe-area-context": "^4.10.5",
-    "react-native-screens": "^3.32.0"
+    "react-native-screens": "^3.32.0",
+    "tsx": "^4.12.0"
   },
   "devDependencies": {
     "@babel/core": "^7.20.0",
+    "@ts-safeql/eslint-plugin": "^3.3.0",
     "@types/node": "^20.14.7",
     "@types/react": "~18.2.45",
-    "eslint-config-upleveled": "^8.3.0",
+    "@types/react-dom": "^18.3.0",
```

Comment:
another error caused by the broken project type detection in `eslint-config-upleveled` -`@types/react-dom` should only be installed in CRA and Next.js apps:

- https://github.com/upleveled/eslint-config-upleveled/blob/6fafffc0012b70e1a54aabdfca000a1e083e1ce4/bin/install.js#L110-L121

Reply:
Removed in this commit [9f15a98](https://github.com/upleveled/expo-example-spring-2024-atvie/pull/4/commits/9f15a98ab7447a2fa94ee7a0592044cf6a497b3f)